test/powershell/engine/Remoting/InvokeCommandRemoteDebug.Tests.ps1

```diff
@@ -0,0 +1,200 @@
+﻿##
+## PowerShell Invoke-Command -RemoteDebug Tests
+##
+
```

Comment:
I can point you to an IoT VM (internally) so you can at least run a smoke test there. IoT and Nano Server are very similar environments, so I would expect it to work there as well.